### ISSUE TITLE
Hologram on ship/Scoreboard/bossbar/hotbar when piloted a ship or on a piloted ship

### ISSUE THREAD

**Issue by TomLewis** [Intybyte/Movecraft-PAPI#1]
Would it be possible to use this somehow with Hologram on ship/Scoreboard/bossbar/hotbar when you piloted a ship for live feedback, or if you boarded another ship thats piloted to gain the same stats.

If this is placeholder API addon, would that mean this would need to be requested via movecraft itself?

**Comment by Intybyte** [comment]
> Would it be possible to use this somehow with Hologram on ship/Scoreboard/bossbar/hotbar when you piloted a ship for live feedback, or if you boarded another ship thats piloted to gain the same stats.

PAPI supports everything you have written, the addon just allows you to use these placeholders, if you want for instance to show a placeholder on the scoreboard then you will need a scoreboard plugin that supports PAPI

> If this is placeholder API addon, would that mean this would need to be requested via movecraft itself?

Every info is collected from interacting with the movecraft plugin, not sure what you are asking.

**Comment by TomLewis** [comment]
> PAPI supports everything you have written, the addon just allows you to use these placeholders, if you want for instance to show a placeholder on the scoreboard then you will need a scoreboard plugin that supports PAPI

haha Yeah, I know how placeholder API works


sometthing has to trigger one of these feedback systems when you pilot a ship, it cant just have a scoreboard showing at all times haha a plugin that listens for when you enter a piloted ship to then shwo you these placeholders.

**Comment by Intybyte** [comment]
you normally use some kind of scripting plugins or make your own addon to use papi, ik a guy hat uses ecoscripts to hook to papi's data or there are also custom TAB & scoreboard plugins that use PAPI, this is out of scope of the plugin in the end

**Comment by TomLewis** [comment]
Yeah I get it, no sweat. I love to use command helper that uses method script, but to would need a movecraft add-on written in java anyway.

I recently started learning java, very slowly 🤣 so I might just see how you grabbed this data from movecraft and see if I can use it directly in a mini movecraft add-on, gives myself another thing to learn! Too many projects not enough time!
